nvme.c

```diff
@@ -8991,17 +8991,7 @@ static int lockdown_cmd(int argc, char **argv, struct command *cmd, struct plugi
 		return -1;
 	}
 
-	struct nvme_lockdown_args args = {
-		.args_size	= sizeof(args),
-		.scp		= cfg.scp,
-		.prhbt		= cfg.prhbt,
-		.ifc		= cfg.ifc,
-		.ofi		= cfg.ofi,
-		.uuidx		= cfg.uuid,
-		.timeout	= nvme_cfg.timeout,
-		.result		= NULL,
-	};
-	err = nvme_lockdown(l, &args);
+	err = nvme_lockdown(l, cfg.scp, cfg.prhbt, cfg.ifc, cfg.ofi, cfg.uuid, NULL);
```

Comment:
Nit: `cfg.uuid` could be renamed as `uidx` to align with the specs (this also needs to be adjusted in libnvme)